Makefile

```diff
@@ -9,6 +9,10 @@ endif
 compile: fmt vet pkg/liiklus/LiiklusService.pb.go ## Compile target binaries
 	go build .
 
+.PHONY: compile-stream-publish
```

Comment:
Don't make the a PHONY target but rather a true dependency (ideally taking into account the windows .exe prefix) on the binary